app/views/cancel_declaration.scala.html

```diff
@@ -16,61 +16,90 @@
 
 @import controllers.routes._
 @import forms.CancelDeclaration
-@import uk.gov.hmrc.play.views.html._
-
-@import views.components.inputs.RadioOption
-@import forms.cancellation.CancellationChangeReason
 @import forms.cancellation.CancellationChangeReason.{NoLongerRequired, Duplication, OtherReason}
 @import forms.Choice
 @import forms.Choice.AllowedChoiceValues.CancelDec
 @import views.Title
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.BackButton
+@import views.html.components.gds._
 
-@this(main_template: views.html.main_template)
+@this(
+        govukLayout: gdsMainTemplate,
+        govukButton: GovukButton,
+        govukRadios: govukRadios,
+        errorSummary: errorSummary,
+        pageTitle: pageTitle,
+        exportsInputText: exportsInputText,
+        exportsInputTextArea: exportsInputTextArea,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(form: Form[CancelDeclaration])(implicit request: Request[_], messages: Messages)
 
-@main_template(
-    title = Title("cancellation.title")
+@govukLayout(
+    title = Title("cancellation.title"),
+    backButton = Some(BackButton(messages("site.backToSelectionPage"), controllers.routes.ChoiceController.displayPage(Some(Choice(CancelDec)))))
 ) {
 
-    @helper.form(action = CancelDeclarationController.onSubmit(), 'autoComplete -> "off") {
-        @helper.CSRF.formField
-
-        @components.back_link(controllers.routes.ChoiceController.displayPage(Some(Choice(CancelDec))))
+    @formHelper(action = CancelDeclarationController.onSubmit(), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
-        @components.error_summary(form.errors)
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
Are you sure we need this `<div>`?   We don't tend to add that on other pages.

Reply:
It was necessary to solve the problem of showing the content in the two-thirds format, as shown in the prototype.